catalogue-service-micro/src/main/java/eu/nimble/service/catalogue/impl/CatalogueController.java

```diff
@@ -737,6 +737,33 @@ public ResponseEntity getCatalogueInSemanticFormat(@ApiParam(value = "uuid of th
         }
     }
 
+    @RequestMapping(value = "/catalogue/{partyId}",
+            produces = {"application/json"},
+            method = RequestMethod.GET)
+    public ResponseEntity getCatalogues(@ApiParam(value = "Identifier of the party for which the catalogue to be retrieved", required = true) @PathVariable String partyId,
```

Comment:
The name of the service is a bit misleading. Could you please rename it as getCatalogueIds or something like that ?

Reply:
Let's have it more self-descriptive: `getAllCatalogueIdsForParty`

Reply:
Sure guys.